msp/compliance/orgs_and_cloud_accounts_report/orgs_and_cloud_accounts_report.pt

```diff
@@ -0,0 +1,280 @@
+name "Orgs and Clouds Vendor Accounts Report Policy"
```

Comment:
Product doesn't want to use the work Report.  Remove **Report Policy**.

Reply:
Done.